.github/workflows/build.yaml

```diff
@@ -312,7 +312,7 @@ jobs:
       AZURE_CLIENT_SECRET: ${{ secrets.AZURE_CLIENT_SECRET }}
       AZURE_TENANT_ID:     ${{ secrets.AZURE_TENANT_ID }}
     needs: build
-    runs-on: windows
+    runs-on: windows-large
```

Comment:
may want to leave this change out

Reply:
? but if it makes the difference between hanging forever and running, we should keep it.

Reply:
yeah, for now seems required to guarantee getting a runner